src/ctia/entity/attack_pattern.clj

```diff
@@ -63,10 +60,9 @@
     (merge
      em/base-entity-mapping
      em/sourcable-entity-mapping
+     em/describable-entity-mapping
      em/stored-entity-mapping
      {:abstraction_level em/token
-      :name em/token
```

Comment:
merging this PR will make CTIA fail to restart due to a forbidden mapping change.